CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+How to Contribute
+=================
+
+Getting help
+------------
+To ask questions about XBee MicroPython go to the [Digi Forum][forum].
+
+
+Reporting a bug
+---------------
+The way to report bugs is to use the [GitHub issue tracker][issues]. Before
+reporting a bug, please read the following points:
+
+1. Make sure that is really a bug by checking the [documentation][doc] and
+   consulting the **Which features apply to my device?** section.
+2. If you still think you have found a bug, make sure someone has not already
+   reported it. See the list of [known issues][issues].
+3. If it has not been reported yet, create a new issue. Make sure to add enough
+   detail so that the bug can be reproduced.
+
+**Note**: The issue tracker is for bugs, not requests for help. Questions
+should be asked on the [Digi Forum][forum] instead.
+
+
+Suggesting a new feature
+------------------------
+1. Consult the **Which features apply to my device?** section of the
+   [documentation][doc] to ensure that the behavior you would like is not
+   excluded in your device.
+2. Make sure someone has not already requested it. See the list of
+   [known issues][issues].
+3. Submit your request in the issue tracker.
+
+
+Contributing code
+-----------------
+1. Fork the [XBee MicroPython][xbee-micropython] repository
+   ([how to fork a repo][fork-repo]).
+2. Create a branch for your contribution. Use a name that defines the purpose
+   of the additions/modifications.
+3. Make your changes following the code style used in the samples. If you
+   are adding a new sample or library, follow the steps described in the
+   [Contributing a new sample](#contributing-a-new-sample) and
+   [Contributing a new library](#contributing-a-new-library) guides.
+4. Execute your code and verify the new functionality or fix works properly.
+5. Submit a pull request ([how to create a pull request][pull-request]).
+
+A project developer will review your work and then merge your request into the
+project, or come back to you with comments and/or questions.
+
+### Contributing a new sample
+
+XBee MicroPython samples are located in the `samples/` directory of the
+repository's root path. Samples are organized in categories, which are 
+directories that group samples by functionality.
+
+You can create and nest categories as needed, but before doing so, make sure
+there is not already a category that meets your needs.
+
+#### Sample contents
+
+* A sample requires a directory with the sample ID as name. This ID must be a
+  short name describing the example purpose, in lowercase and without blank
+  spaces. 
+* Each sample must include at least the following files inside its directory:
+  * `README.md` This file describes the sample functionality, requirements,
+    setup, etc. and is used by the **XBee MicroPython plug-in for PyCharm**
```

Comment:
I would use "plugin" instead of "plug-in" (https://www.jetbrains.org/intellij/sdk/docs/basics.html).

Reply:
Done.